### ISSUE TITLE
Absolute IRI "sec:BbsBlsSignature2020" confused with prefix "sec".

### ISSUE THREAD

**Issue by OR13** [mattrglobal/jsonld-signatures-bbs#93]
```
   "@mattrglobal/jsonld-signatures-bbs": "0.7.1-unstable.2673a0a",
```

I believe this is caused by the interaction between sec-v3-unstable...

https://github.com/mattrglobal/jsonld-signatures-bbs/blob/2673a0a077c232ca5be61b93339be547e5341635/src/BbsBlsSignature2020.ts#L54

imo this call to super should be with https://github.com/mattrglobal/jsonld-signatures-bbs/blob/2673a0a077c232ca5be61b93339be547e5341635/src/BbsBlsSignature2020.ts#L401

See similar approach taken here:

https://github.com/w3c-ccg/lds-jws2020/blob/master/packages/json-web-signature-2020/src/JsonWebSignature.ts#L25

**Comment by tplooker** [comment]
@OR13 can you provide an example of when this is occuring because I believe this is correct behaviour based on the limitations of the security v2 context

**Comment by kdenhartog** [comment]
This is a work around because the new sec-v3 context doesn't properly expand with the current json-ld signatures implementation due to the removal of the prefixes and moving to long form IRIs. See this issue for additional details: https://github.com/digitalbazaar/jsonld-signatures/issues/118

**Comment by OR13** [comment]
more details here: https://github.com/w3c-ccg/security-vocab/pull/80

**Comment by OR13** [comment]
I am attempting to get the credentials to work with only:

- credentials/v2 
- security/v3-unstable

**Comment by OR13** [comment]
Reminder to fix this :
https://github.com/mattrglobal/jsonld-signatures-bbs/blob/2673a0a077c232ca5be61b93339be547e5341635/src/BbsBlsSignature2020.ts#L59

Then we won't need to load sec-v3 with when cred-v2 or bbs-v1 is included.

**Comment by OR13** [comment]
IMO, we should just wait until the latest stable uses addresses this issue 

Then add examples of linked data signatures and credentials, that replace the current sample and only use:

- credentials/v2 
- security/v3
- credentials/v1  + https://w3c-ccg.github.io/ldp-bbs2020/contexts/v1 
- security/v2 + https://w3c-ccg.github.io/ldp-bbs2020/contexts/v1

**Comment by tplooker** [comment]
This issue will be addressed in the up and coming v0.8.0 release
